recipes/ftfy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ftfy" %}
+{% set version = "4.4.2" %}
```

Comment:
Yes, I'm playing safe adding the minimum version required and then will update to the latest `5.x`. Also, `ftfy 5.x` is py3 only.